src/server/models/functions/updateLinkStatistics.ts

```diff
@@ -0,0 +1,67 @@
+import { Url } from '../url'
+import { Clicks } from '../statistics/daily'
+import { Devices } from '../statistics/devices'
+import { WeekdayClicks } from '../statistics/weekday'
+
+/**
+ * Name of the link statistics update SQL function.
+ */
+export const rawFunctionName = 'update_link_statistics'
```

Comment:
can we move this file into the service instead, so that everything is in the same place?

also i'm not sure if the `rawFunctionName` abstraction is really necessary - it doesn't appear to add value since there is only one function.

Reply:
Initially, I abstracted out the function name because if there is any disparity between the initialised SQL function's name and the function name called when updating link statistics, we will only be able to catch it in runtime. But since there is only one function, I guess we can not abstract it. Additionally, now that I think of it, hardcoding it might also improve performance with lesser string interpolation.

Reply:
For now, I will also be shifting the file to the repository, which should help put everything in one place, and ensures better consistency between the function initialisation and calls.